.gitignore

```diff
@@ -0,0 +1,4 @@
+.eggs
```

Comment:
If it's a folder (which it looks like) it should be written as `.eggs/` to prevent false-positives.  Also, if you only want to ignore the one in the top-level folder (likely) you probably want `/.eggs/`.